test/unit/math/Ray.js

```diff
@@ -274,8 +279,38 @@ test( "distanceSqToSegment", function() {
 	ok( ptOnSegment.distanceTo( one3 ) < 0.0001, "Passed!" );
 	ok( ptOnLine.distanceTo( one3 ) < 0.0001, "Passed!" );
 	ok( distSqr < 0.0001, "Passed!" );
-});
+	// test cases from
+	// http://www.geometrictools.com/Documentation/DistanceLine3Line3.pdf
+
+	// Example 1 An example where the minimum point is interior but some
+	// numerical rounding errors lead to an inaccurate result.
+
+	var P0 = new THREE.Vector3(-1.0264718499965966,	  9.6163341007195407e-007, 0.0);
+	var P1 = new THREE.Vector3( 0.91950808032415809, -1.0094441192690283e-006, 0.0);
+	var Q0 = new THREE.Vector3(-1.0629447383806110,	  9.2709540082141753e-007, 0.0);
+	var Q1 = new THREE.Vector3( 1.0811583868227901,	 -1.0670017179567367e-006, 0.0);
+
+	rayP = new THREE.Ray( P0, P1);
```

Comment:
The second argument in the `Ray` constructor is a direction vector, and must have unit length (be normalized). Where is `rayP` used?
This example appears to be about the distance between two line segments, which is a different problem.